### DISCUSSION TITLE
How to remove erosion damage from REMBG using Gimp

### DISCUSSION THREAD

**Discussion by LinuxBeaver** [danielgatis/rembg#146] [General]
1. Alpha to select by alt clicking a thumbnail

2. Invert Selection

3. Apply repeats of Gimp's color to alpha filter set to black. This successfuly removes erosion damage caused by REMBG's default setting.

Example here

https://streamable.com/zpp5gh

**Comment by chameroy** [comment]
Is the alpha matting supposed to prevent erosion damage? I'm asking because I don't understand how the matting works.

**Comment by tsukimine** [comment]
how can i use it in gimp on ubuntu mate, help me please
